Tools/autotest/arducopter.py

```diff
@@ -67,6 +61,15 @@ def __init__(self, binary,
         self.sitl = None
         self.hasInit = False
 
+    def mavproxy_options(self):
+        ret = super(AutoTestCopter, self).mavproxy_options()
+        if self.frame != 'heli':
+            ret.append('--quadcopter')
+        return ret
+
+    def sitl_streamrate(self):
+        return 5
```

Comment:
Do you know the reason for having different streamrates? I would just run them all equal.